Modules/_cursesmodule.c

```diff
@@ -1831,62 +1831,71 @@ PyCursesWindow_GetStr(PyObject *op, PyObject *args)
     PyCursesWindowObject *self = _PyCursesWindowObject_CAST(op);
 
     int x, y, n;
-    char rtn[1024]; /* This should be big enough.. I hope */
-    int rtn2;
+    int rtn;
+
+    /* could make the buffer size larger/dynamic */
+    const int max_buf_size = 2048;
```

Comment:
```suggestion
    Py_ssize_t max_buf_size = 2048;
```

No need for `const` qualifiers, they usually make the code a bit less readable.